### DISCUSSION TITLE
which measured value makes sense?

### DISCUSSION THREAD

**Discussion by sat-fan2019** [jeroenterheerdt/HAsmartirrigation#368] [Q&A]
My weather station offers temperature wind / air pressure / UV / rain / dew point.

But the question is what makes sense in each case (max/average/first/last/sum etc.
Thx for help !

**Comment by jeroenterheerdt** [comment]
Have you read the readme?

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: sat-fan2019 ***@***.***>
Sent: Wednesday, May 15, 2024 10:38:10 PM
To: jeroenterheerdt/HAsmartirrigation ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jeroenterheerdt/HAsmartirrigation] which measured value makes sense? (Discussion #368)


My weather station offers temperature wind / air pressure / UV / rain / dew point.

But the question is what makes sense in each case (max/average/first/last/sum etc.
Thx for help !
